ert_storage/database_schema.py

```diff
@@ -49,6 +51,17 @@ class Experiment(Base):
     )
     name = sa.Column(sa.String)
     ensembles = relationship("Ensemble", foreign_keys="[Ensemble.experiment_id]")
+    observations = relationship(
+        "Observation", foreign_keys="[Observation.experiment_id]"
```

Comment:
So a single experiment has immutable observations, ie. removing an observation would cause creation a new experiment then?

Reply:
I think that should be the case :)